aikau/src/test/resources/alfresco/dashlets/InfiniteScrollDashletTest.js

```diff
@@ -42,61 +42,51 @@ define(["alfresco/TestCommon",
       },
 
       "Scroll to bottom of first dashlet body": function() {
-         // Click on the first row to give it focus...
-         return browser.findByCssSelector("#INFITE_SCROLL_LIST_1 tr:nth-child(1) .alfresco-renderers-Property")
-            .click()
-            .pressKeys(keys.ARROW_DOWN)
-            .pressKeys(keys.ARROW_DOWN)
-            .pressKeys(keys.ARROW_DOWN)
-            .pressKeys(keys.ARROW_DOWN)
-            .pressKeys(keys.ARROW_DOWN)
-            .pressKeys(keys.ARROW_DOWN)
-            .pressKeys(keys.ARROW_DOWN)
-            .pressKeys(keys.ARROW_DOWN)
-         .end()
-         .getLastPublish("BELOW_ALF_EVENTS_SCROLL")
+         return browser.execute(function() {
+               document.querySelector("#BELOW_DASHLET .alfresco-dashlets-Dashlet__body__widgets").scrollTop += 20;
```

Comment:
I know you said that there was a bug with the keyboard navigation, but I'd like to keep the use of the keyboard navigation to ensure that it triggers the reload and that it doesn't reset the focused row back to the first item again.